api/challenge_api/app/jobs/application_job.rb

```diff
@@ -0,0 +1,2 @@
+class ApplicationJob < ActiveJob::Base
```

Comment:
I'm unfamiliar with ActiveJob, what role do you envision this class will play in the app?

Reply:
I would only envision this being useful if we needed to use something like Redis or RabbitMQ to cache data. This is the only way I've used ActiveJob in the past, am I correct in assuming that's beyond the scope of the project?

Reply:
I like RabbitMQ, Redis and job queues, but I can't think of any way to implement in this app without the functionality sticking out like a sore thumb. This app won't ingest nearly enough data and requests to require the use of a job queue.